contracts/PrivateContract.sol

```diff
@@ -1,38 +1,72 @@
-//! Private Transactions State Storage
-//! By Parity Team, 2017.
-//!
-//! This contract keeps track of a private transaction state (supposedly encrypted)
-//! and allows its change only if all the Validators (from a static list, initialized in constructor)
-//! have signed a new state (hashed together with a current nonce, for replay protection).
+// Private Transactions State Storage
+//
+// This contract keeps track of a private transaction state (supposedly encrypted)
+// and allows its change only if all the Validators (from a static list, initialized in constructor)
+// have signed a new state (hashed together with a current nonce, for replay protection).
+//
+//
```

Comment:
I would put the license header before the previous block comment describing the contract.